assets/scss/lib/ck_style.scss

```diff
@@ -1,17 +1,11 @@
 /* Imports all styles needed to be included within the site's CK editor instances. */
 
-// Import council theme settings
-@import 'theme'; // replace with '_variables-REPLACEWITHYOURCOUNCILNAME'
```

Comment:
Sorry Chris, I missed this one last time. Is this not needed at all?

Reply:
@Adnan-cds no ive removed it because when we first put together the original croydon site, the one localgov was based on, we added this 'theme.scss' in knowing it would become a base shareable theme and thinking it would be a central place for users to put all of their modifiers when customising the base theme. However, thats not the route we have taken... modifiers are added to partials or duplicates of existing partials.

Reply:
Got it. Thanks for explaining.